src/component/Card/WishListCard.jsx

```diff
@@ -0,0 +1,80 @@
+import "./wishListCard.css";
+import "./card.css";
+import "../index";
+import { useWishList } from "../Context/WishListContext/WishListContext";
+
+const WishListCard = ({ wishlist }) => {
```

Comment:
Suggestion: You could destructure wishlistem and wishListCount here itself(line:6)

Reply:
If i am doing that my code is breaking thats why i destructed it outside